demos/smart_classroom_demo/cpp/src/cnn.cpp

```diff
@@ -13,68 +13,74 @@
 
 #include <inference_engine.hpp>
 
+#include <map>
+
+#include "openvino/core/layout.hpp"
+#include "openvino/openvino.hpp"
+
 CnnDLSDKBase::CnnDLSDKBase(const Config& config) : config_(config) {}
 
 void CnnDLSDKBase::Load() {
-    auto cnnNetwork = config_.ie.ReadNetwork(config_.path_to_model);
-
+    auto cnnNetwork = config_.ie.read_model(config_.path_to_model);
 
-    const int currentBatchSize = cnnNetwork.getBatchSize();
-    if (currentBatchSize != config_.max_batch_size)
-        cnnNetwork.setBatchSize(config_.max_batch_size);
+    const int currentBatchSize = cnnNetwork->input().get_shape()[0];
+    if (currentBatchSize != config_.max_batch_size) {
+      cnnNetwork->get_parameters()[0]->set_layout("NCHW");
```

Comment:
You set it later with `ov::preprocess::PrePostProcessor`. Just put `ov::set_batch(cnnNetwork, config_.max_batch_size);` after `ov::preprocess::PrePostProcessor` usage

Reply:
Done